packages/lib/src/core/Analytics/CAEventsQueue.ts

```diff
@@ -0,0 +1,72 @@
+import { HttpOptions, httpPost } from '../Services/http';
+import { AnalyticsObject, EventQueueProps } from './types';
+
+interface CAActions {
+    channel: 'Web';
+    events: AnalyticsObject[];
+    errors: AnalyticsObject[];
+    logs: AnalyticsObject[];
+}
+
+export interface EQObject {
+    add: (t, a) => void;
+    run: (id) => Promise<any>;
+    getQueue: () => CAActions;
+    _runQueue: (id) => Promise<any>;
+}
+
+const CAEventsQueue = ({ analyticsContext, clientKey, analyticsPath }: EventQueueProps) => {
```

Comment:
What does `CA` stands for? `CheckoutAdyen`? 
(Every time I read it I think it is `CustomerArea` 😅  )

I find it very hard to read and understand the types/objects when they get abbreviated like that (CAActions, EQObject, and so on). We don't have guidelines for that, but I'd rather try to not abbreviate that much or try to be more explicit. I believe it improves the readability of the codebase

Reply:
CA stands for CheckoutAnalytics of course! :-D
However I only added the "CA" to distinguish it from the existing EventsQueue. Now that is gone we can drop the prefix